.editorconfig

```diff
@@ -0,0 +1,13 @@
+root = true
+
+charset = utf-8
+
+end_of_line = lf
+
+insert_final_newline = true
+
+[!*.{py, md, yml}]
+indent_style = tab
+
+[!*.{py, md, yml}]
+indent_size = 4
```

Comment:
While this specifies the format for this glob, there isn't any specification for files that match those extensions. Basically a corresponding
```
[*.{py,md,yml}]
indent_style = space
indent_size = 2
```